api/scpca_portal/models/project.py

```diff
@@ -77,6 +79,10 @@ def computed_files(self):
     def input_data_path(self):
         return common.INPUT_DATA_PATH / self.scpca_id
 
+    @property
+    def input_merged_data_path(self):
+        return self.input_data_path / "merged"
```

Comment:
return self.input_data_path / f'{self.scpca_id}_merged.rds'?

Reply:
This gets used on line 95, so it is just used to break up where we source the "merged" files, meaning celltype-report and the actual data in the input folder.